actionview/lib/action_view/helpers/debug_helper.rb

```diff
@@ -5,32 +5,71 @@
 module ActionView
   # = Action View Debug Helper
   #
-  # Provides a set of methods for making it easier to debug Rails objects.
+  # Provides a method to debug Rails objects.
```

Comment:
If we're going to touch this line, we can move from `Rails` to `Ruby` here. There is nothing such a `Rails object` AFAIK.

Reply:
I removed "Rails" entirely, if we're in Rails I think we can't debug any other objects than Ruby objects. wdyt?